### PR TITLE
List endpoints

### PR DESCRIPTION
**Pull request by jaime-human**

### CODE REVIEW

File: src/components/list/list.service.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * Post list
+ * @param {Object} list accountId, isDefault, name, isPublic & mediaItems
+ * @throws {Unauthorized} When credentials are wrong or not provided
+ * @throws {InternalServerError} When there's an unhandled error.
+ */
+module.exports.postList = async (list) => {
+    list.accountId = mongoose.Types.ObjectId(list.accountId);
+
+    listModel.create(list);
```
Comment (pawap90) [inline src/components/list/list.service.js:15]: Missing `await`. It should be `await listModel.create(list);`

File: src/components/list/list.service.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const listModel = require('./list.model');
+const mongoose = require('mongoose');
+
+/**
+ * Post list
+ * @param {Object} list accountId, isDefault, name, isPublic & mediaItems
+ * @throws {Unauthorized} When credentials are wrong or not provided
```
Comment (pawap90) [inline src/components/list/list.service.js:9]: This is not true on this level. The method doesnt throw Unauthorized, the middleware takes care of that earlier on the request.

File: documentation/list.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                imagePath: "willthisgetanimage.com/image.png",
+                genres: ["Drama", "Philossophy", "Psychedelich"],
+                year: 1998
+              }
+            ]
+      responses:
+        200:
+          description: List was created succesfuly
+        401:
+          description: Unclear
```
Comment (pawap90) [inline documentation/list.yaml:49]: The description here could be something like: `Authorization token was not provided`

File: documentation/list.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        500:
+          description: Unclear
```
Comment (pawap90) [inline documentation/list.yaml:51]: This description could be `There was an unexpected error creating the list`

File: documentation/list.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+      tags:
+        - List
+      summary: Gets users lists
+      produces:
+        - application/json
+      responses:
+        200:
+          description: List were obtained succesfuly
+        401:
+          description: Unclear
```
Comment (pawap90) [inline documentation/list.yaml:64]: Add something like `Authorization token not provided`

File: documentation/list.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        500:
+          description: Unclear
```
Comment (pawap90) [inline documentation/list.yaml:66]: Something like `Unexpected error retrieving the user's lists`

**Comment by pawap90** [comment]
Closes #13 
Closes #14

**Comment by jaime-human** [comment]
Thanks for your comments! I've already applied those changes.
Greetings from Argentina.

**Pull request merged**